Sources/Extensions/UIKit/UIViewExtension.swift

```diff
@@ -98,4 +98,31 @@ extension UIView {
 
 }
 
+// MARK: - Constraints
+
+@available(iOS 9.0, *)
+extension UIView {
+
+    public func addConstraints(to edges: UIRectEdge = .all, insets: UIEdgeInsets = .zero) {
+        guard let superview = superview else {
+            return
```

Comment:
I don't know about you, but personally I feel like if a superview was not found, crashing with an error message could be more preferable than silently doing nothing. Trying to add constraints to a view with no superview is most likely a mistake, and people should be alerted, right?

Reply:
Agree, I hesitated about this, so I will change it. Thanks for the feedback